Doc/library/enum.rst

```diff
@@ -591,6 +591,9 @@ However, they still can't be compared to standard :class:`Enum` enumerations::
     [0, 1]
 
 
+Note that the :class:`IntEnum` class is defined by mixing in :class:`int` to :class:`Enum`, which has some surprising
```

Comment:
This violates our style by describing an intentional design feature as surprising or dangerous.  Instead, the devguide states that docs be written affirmatively to show what the tool actually does and how to access its features.

For example, this would be an improvement:

    Enumeration support several different display styles:

        >>> repr(Color.red)
        '<Color.red: 1>'
        >>> str(Color.red)
        'Color.red'
        >>> format(Color.red)
       '1'

    These three styles can be accessed in f-string as follows:

        print(f'{Color.red!r} or {Color.red!s} or {Color.red}' )